### DISCUSSION TITLE
Apple Silicon native support

### DISCUSSION THREAD

**Discussion by tfagundess** [UniversalMediaServer/UniversalMediaServer#2989] [Ideas]
Searched here and in the forums but I see no mention on that. Is it a planned feature? It would be awesome to have UMS as a Universal App for the "new" M1 machines. Thanks!

Edit: adding link.

https://developer.apple.com/programs/universal/

**Comment by SubJunk** [comment]
@tfagundess I don't know what a `Universal App` is, but I know some of our dependencies are being released for M1 chips, so we could make at least a partial M1 release. What's a Universal App?

**Reply by tfagundess** [reply]
Universal App is what Apple calls an application that is both native for Intel and Apple Silicon. It’s some sort of wrapper that lets you have only one executable that automatically chooses the correct native code to run.

About the dependencies, for sure they would need to be M1 compatibles, yes.

edit: added a link to the original post.

**Reply by SubJunk** [reply]
This will be quite a big job but I think it will be worth it. We have an issue for it now too https://github.com/UniversalMediaServer/UniversalMediaServer/issues/3009

**Reply by tfagundess** [reply]
Nice, thanks!
